omnibus_build.sh

```diff
@@ -35,6 +37,12 @@ if [ -n "$RPM_SIGNING_PASSPHRASE" ]; then
   gpg --import /keys/RPM-SIGNING-KEY.private
 fi
 
+git --git-dir=/integrations-core fetch --all || true
+git --git-dir=/integrations-core checkout $INTEGRATION_CORE_BRANCH || true
```

Comment:
I don't understand why we're fetching checking out the `integrations-core` repo here, we should probably remove these lines no?